task/reindex.go

```diff
@@ -194,8 +194,9 @@ func uriProducer(ctx context.Context, tracker *Tracker, errorChan chan error, z
 			return
 		}
 		for _, item := range items {
-			if strings.Contains(item.URI, "/timeseries/") && strings.Contains(item.URI, "/previous/") {
-				log.Info(ctx, "not indexing uri as previous timeseries", log.Data{
+			// Exclude previous versions of any release/timeseries (e.g., v1, v2, etc.)
+			if strings.Contains(item.URI, "/previous/") {
+				log.Info(ctx, "not indexing uri as item is a previous version", log.Data{
```

Comment:
I would say 'release' here for clarity and consistency:
```suggestion
				log.Info(ctx, "not indexing uri as release is a previous version", log.Data{
```